oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/multiline_textarea.less

```diff
@@ -0,0 +1,86 @@
+/*******************************
+            Theme
+*******************************/
+
+/*
+  The type is the type of the component as defined in semantic-ui - element, collection, view, module
+  For simple component, 'element' is the correct type
+*/
+@type: "element";
+
+/*
+  This is the name of the component - though not required, keep it the same as the filename
+*/
+@element: "multiline_textarea";
+
+/*******************************
+
+    DO NOT FORGET !!!
+
+Edit *theme.config* in your site 
+and add (replace 'sample' with the
+name of the component from above)
+
+@sample : 'default';
+
+*******************************/
+
+@import (multiple) "../../theme.config";
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+*/
+
+// HTMl+CSS trick, according to: https://stackoverflow.com/a/47896872
+
+.textarea-container {
+  position: relative;
+  width: 100%;
+  z-index: 1;
+}
+
+.ui.multiline-textarea {
+  resize: none;
+  min-height: 38px; // 38px is the default height of a textarea
+  border: 1px solid @borderColor;
+  border-radius: .28571429rem 0 0 .28571429rem;
+  font-family: inherit;
+  padding: 0.5em 1em;
+  padding-right: 2.7em;
+  width: 100%;
+  outline: none;
+  background: white;
+
+  &:focus + .textarea-overlay {
+    display: none;
+  }
+}
+
+.textarea-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  pointer-events: none;
+
+  // Match textarea styling
+  min-height: 38px;
```

Comment:
I'd define following properties in a single css rule block for both .multiline-textarea and .textarea-overlay to avoid duplication